protobufs/canvas_generated/messages/effects.proto

```diff
@@ -163,6 +163,8 @@ enum EffectType {
   COMMIT_CLOSE_GOAL_COMMAND = 131;
   ENTER_IN_ERROR_CLOSE_GOAL_COMMAND = 132;
 
+  CREATE_QUESTIONNAIRE_RESULT = 138;
```

Comment:
Ensure that this change does not conflict with other gRPC interface changes that are part of the same release.